src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -44,9 +46,12 @@ public function configureOptions(OptionsResolver $resolver)
                     return null;
                 }
 
+                $this->locale = $options['locale'];
```

Comment:
It's either this, either decorating the `CountryType`'s `ChoiceLoaderInterface` implementation when using the `locale` option, so we keep a cache of base choice list (and will always be used as before when not using the option). But this cache is not worth it to me.